### PR TITLE
feat: add issue templated for github issues

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Inspired by https://github.com/liferay/liferay-frontend-guidelines/pull/82

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: 🐛 Bug
+about: Issues for filing a Bug
+labels: bug
```
Comment (matuzalemsteles) [inline .github/ISSUE_TEMPLATE/Bug.md]: Just a question, should we add the `type: bug` label here to reference our existing labels? or he had just created a new label.

Comment (bryceosterhaus) [reply .github/ISSUE_TEMPLATE/Bug.md]: This is the GH syntax for picking up existing labels. 

You can check them out actually working on my local repo at https://github.com/bryceosterhaus/clay/issues/new/choose

**Comment by bryceosterhaus** [comment]
@matuzalemsteles instead of creating a Feature Request one, I think the Proposal issue can cover that. Its a bit more generic to allow for other types of proposals as well.

**Comment by matuzalemsteles** [comment]
> @matuzalemsteles instead of creating a Feature Request one, I think the Proposal issue can cover that. Its a bit more generic to allow for other types of proposals as well.

Oh ok, I get the thought of creating a proposal, because I already have something defined and feature request I don't want to think of something, but it's just an idea... but feel free to merge.

**Comment by bryceosterhaus** [comment]
@matuzalemsteles lets start with this for now and see how people use it. We can always add more as we see more use cases.

**Comment by wincent** [comment]
💍  I love this one.

**Pull request merged**